Cod3rsGrowth.Infra/Repositorios/RepositoryClube.cs

```diff
@@ -30,7 +31,8 @@ public Clube ObterPorId(int id)
 
         public int Criar(Clube objeto)
         {
-            throw new NotImplementedException();
+            database.Insert(objeto);
```

Comment:
acredito que o insert ja te retorne o id, veja isso. se for o caso, mude o return desse metodo de lugar